insecure/conduitMaster.go

```diff
@@ -5,6 +5,9 @@ import (
 	"github.com/onflow/flow-go/network"
 )
 
+// CorruptedFactoryPort defines the port on which attacker dials a corruptible conduit factory.
+const CorruptedFactoryPort = 5000
+
```

Comment:
not specific to this line, but a general comment. Let's use a more inclusive word Instead of `master`. Maybe `Coordinator` or `Controller`?

Reply:
applied in https://github.com/onflow/flow-go/pull/2144/commits/7cca5cab58bd76956d2ca057beedc6eaa032fbf9